products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp

```diff
@@ -273,9 +277,9 @@ namespace app::home_screen
         return greetingCollection[dist(*rngEngine)];
     };
 
-    void HomeScreenPresenter::setUSBStatusConnected()
+    void HomeScreenPresenter::updateUsbStatus()
     {
-        getView()->setUSBStatusConnected();
+        getView()->updateUsbStatus(usbStatusModel.isUsbConnected(batteryModel.getLevelState().state));
```

Comment:
I'd split this to just simplify read this line
```suggestion
        const auto batteryState = batteryModel.getLevelState().state;
        const auto usbStatus = usbStatusModel.isUsbConnected(batteryState);
        getView()->updateUsbStatus(usbStatus);
```